backend/config.js

```diff
@@ -9,9 +9,7 @@ module.exports = {
     tokenRefreshThresholdSeconds: process.env.TOKEN_REFRESH_THRESHOLD_SECONDS || 60,
     url: process.env.PRISON_STAFF_HUB_UI_URL || `http://localhost:${process.env.PORT || 3002}/`,
     maximumFileUploadSizeInMb: process.env.MAXIMUM_FILE_UPLOAD_SIZE_IN_MB || 200,
-    featureFlags: {
```

Comment:
was getting a weird react error around this so moved out of feature blog (not used by react anyway)